applications/MappingApplication/custom_searching/mpi/interface_communicator_mpi.cpp

```diff
@@ -129,6 +130,8 @@ void InterfaceCommunicatorMPI::FinalizeSearchIteration(const MapperInterfaceInfo
                                                                mCommRank,
                                                                mMapperInterfaceInfosContainer);
 
+    mMeshesAreConforming = ParallelEnvironment::GetDefaultDataCommunicator().MinAll(mMeshesAreConforming);
```

Comment:
note that this can be problematic if we use subcomms
But I haven't figured out yet which comm to use in the rest of the app so I guess its fine :)

Reply:
Yep that's why I told you that I was not sure if we should put this inside the mapper itself. I see no problem in removing this and returning the local value.

The user then should sync it outside, but I think that's perfectly reasonable